### PR TITLE
YLP-2381 As a HCP, I want to manage patient invites made by other members of the teams

### PR DESCRIPTION
**Pull request by TimCanu**
…bers of the teams

### CODE REVIEW

File: api/invite.go [hunk 1]
```diff
@@ -180,6 +180,58 @@ func (a *Api) GetReceivedInvitations(res http.ResponseWriter, req *http.Request,
...
+	tokenValue := getSessionToken(req)
+	token := a.token(res, req)
+	if token == nil {
+		return
+	}
+
+	teamID := vars["teamId"]
+	patientID := vars["patientId"]
+
+	_, _, err := a.getTeamForUser(nil, tokenValue, teamID, token.UserId, res)
```
Comment (CedricPerotto) [inline api/invite.go]: We should use the context if available
```suggestion
	_, _, err := a.getTeamForUser(req.Context(), tokenValue, teamID, token.UserId, res)
```

File: api/invite.go [hunk 2]
```diff
@@ -180,6 +180,58 @@ func (a *Api) GetReceivedInvitations(res http.ResponseWriter, req *http.Request,
...
+		Type:   models.TypeMedicalTeamPatientInvite,
+		UserId: patientID,
+		Status: models.StatusPending,
+		Team: &models.Team{
+			ID: teamID,
+		},
+	}
+
+	var pendingInvite *models.Confirmation
+	pendingInvite, err = a.Store.FindConfirmation(req.Context(), &inviteToFind)
```
Comment (CedricPerotto) [inline api/invite.go:224]: This error is not handled
```
	pendingInvite, err = a.Store.FindConfirmation(req.Context(), &inviteToFind)
	if err !=nil {
	....
```

File: clients/hydrophone/hydrophone_client.go [hunk 3]
```diff
@@ -151,6 +153,33 @@ func (client *Client) GetSentInvitations(ctx context.Context, userID string, aut
 	return nil, handleErrors(res, req)
 }
 
+func (client *Client) GetPatientTeamPendingInvite(ctx context.Context, teamId string, patientId string, authToken string) (*models.Confirmation, error) {
+	logger := appContext.GetLogger(ctx)
+	req, err := client.getFullRequestWithContext(ctx, "GET", authToken, nil, map[string]string{}, "teams", teamId, "patients", patientId, "invite")
+	if err != nil {
+		return nil, errors.Wrap(err, "GetSentInvitations: error formatting request")
```
Comment (CedricPerotto) [inline clients/hydrophone/hydrophone_client.go]:
```suggestion
		return nil, errors.Wrap(err, "GetPatientTeamPendingInvite: error formatting request")
```

File: go.mod [hunk 4]
```diff
@@ -7,68 +7,53 @@ require (
 	github.com/gorilla/mux v1.8.0
 	github.com/k3a/html2text v1.1.0
 	github.com/mdblp/crew v1.4.4
-	github.com/mdblp/go-common v1.7.0
-	github.com/mdblp/go-common/v2 v2.0.0
+	github.com/mdblp/go-common v1.9.0
+	github.com/mdblp/go-common/v2 v2.0.0-RC4
```
Comment (CedricPerotto) [inline go.mod]: It should depend on a non RC version (i.e 2.0.0  wihch is not really released...)

File: CHANGELOG.md [hunk 5]
```diff
@@ -3,6 +3,10 @@
 Hydrophone is the module responsible for sending emails.
 This API sends notifications to users for things like forgotten passwords, initial signup, and invitations.
 
+## 1.15.0-rc1 - 2023-05-24
```
Comment (CedricPerotto) [inline CHANGELOG.md]: Bump a version

**Review by CedricPerotto — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 5 above)
Comment (TimCanu) [reply CHANGELOG.md]: What do you mean? I bumped the minor, should I bump the major? :thinking:

Comment (CedricPerotto) [reply CHANGELOG.md]: Don't bump RC when merging

Comment (TimCanu) [reply CHANGELOG.md]: Yeah, I won't keep the RC when merging don't worry :slightly_smiling_face:

**Review by CedricPerotto — APPROVED** [review]

**Pull request merged**